src/mui/input/EmbeddedArrayInput.js

```diff
@@ -0,0 +1,249 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'redux-form';
+import inflection from 'inflection';
+
+import FlatButton from 'material-ui/FlatButton';
+import TextFieldLabel from 'material-ui/TextField/TextFieldLabel';
+import ContentCreateIcon from 'material-ui/svg-icons/content/create';
+import ActionDeleteIcon from 'material-ui/svg-icons/action/delete';
+import Divider from 'material-ui/Divider';
+
+import translate from '../../i18n/translate';
+
+import EmbeddedArrayInputFormField from '../form/EmbeddedArrayInputFormField';
+
+const styles = {
+    container: {
+        padding: '0 1em 1em 0em',
+        width: '90%',
+        display: 'inline-block',
+    },
+    innerContainer: {
+        padding: '0 1em 1em 1em',
+        width: '90%',
+        display: 'inline-block',
+    },
+    labelContainer: {
+        padding: '1.2em 1em 0 0',
+        width: '90%',
+        display: 'inline-block',
+    },
+    label: {
+        top: 0,
+        position: 'relative',
+        textTransform: 'capitalize',
+    },
+    removeButton: {
+        float: 'right',
```

Comment:
Using float for this button make it appear **below** the `Divider`, hence do not *link* it to the correct element. Moreover, having the button at the far right make the UI difficult to grasp on large screens.

![screen](http://i.imgur.com/kpKsQBK.png)